src/main/frontend/fs/sync.cljs

```diff
@@ -193,6 +193,11 @@
 
 ;;; ### configs ends
 
+(defn- exp
+  [x]
+  (when (instance? ExceptionInfo x) x))
+
```

Comment:
use this simple fn to simplify the following codes' exception handling.
to avoid write a lot code like: `(if (instance? ExceptionInfo x) x :else-do-something)`.
maybe another better way is to define an exp-monad let, e.g.
```
(let-e [r1 (fn-may-return-exp1)
        r2 (fn-may-return-exp2)] ;; <- early return exp if r1 or r2 is a exception
  (fn3 r1 r2))
```